docs/diagrams/DetailedModelClassDiagram.puml

```diff
@@ -19,7 +19,7 @@ Class Address
 Class Name
 Class Tag
 
-Class Task
+Abstract Class Task
```

Comment:
I think you forgot to save the updated diagram

Reply:
Oh good catch! I will upload the diagram now.